service/service_integration_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func newContainer(t *testing.T) *container.Container {
+func newIntegrationContainer(t *testing.T) *container.DockerContainer {
```

Comment:
It should be better to return the interface instead of the current implementation 
```
func newIntegrationContainer(t *testing.T) container.Container {
```

Reply:
done